resource/html/front/errors.js

```diff
@@ -1,21 +1,27 @@
 const errorCodes = new Map([
-	['Wallet-0001', 'A wallet with the same nickname already exists on your computer'],
-	['Wallet-0002', 'Wrong wallet password'],
-	['Wallet-0003', 'Error while retrieving the specified wallet'],
-	['Wallet-1001', 'A wallet cannot be created without nickname'],
-	['Wallet-1002', 'A wallet cannot be created without password'],
-	['Wallet-1003', 'A technical issue occured while creating the wallet'],
-	['Wallet-2001', 'Specify the wallet wallet address to delete'],
-	['Wallet-2002', 'Error while trying to delete the wallet, may be opened by an other program'],
-	['Wallet-3001', 'Error while trying to import the wallet file'],
-	['Wallet-4001', 'Error while trying to fetch all your wallets'],
+	['Wallet-0001', 'That nickname is taken. Try Another'],
```

Comment:
@SlnPons can you have a look at that plz ?

Reply:
Same here, not just this error but can you have a look at all the error messages ?